kinematics/kinematics.c

```diff
@@ -111,39 +110,29 @@ void kinematics_get_rpm(struct kinematics kin, struct velocity target_vel, rt_in
     // rear-right motor
     cal_rpm.motor4 = x_rpm - y_rpm + tan_rpm;
 
-    if(kin.k_base == TWO_WD)
+    switch(kin.k_base)
     {
+    case TWO_WD:
         res_rpm[0] = cal_rpm.motor3;
         res_rpm[1] = cal_rpm.motor4;
-    }
-    else if(kin.k_base == FOUR_WD)
-    {
-        res_rpm[0] = cal_rpm.motor1;
-        res_rpm[1] = cal_rpm.motor2;
-        res_rpm[2] = cal_rpm.motor3;
-        res_rpm[3] = cal_rpm.motor4;
-    }
-    else if(kin.k_base == ACKERMANN)
-    {
+        rt_memcpy(rpm, &res_rpm[0], sizeof(res_rpm));

```

Comment:
Given that all cases have the same snippet, it can be further simplified

Reply:
ok